plutus-tx-plugin/test/StdLib/Spec.hs

```diff
@@ -66,8 +72,8 @@ testDivMod :: Property
 testDivMod = Hedgehog.property $ do
     let gen = Gen.integral (Range.linear (-10000) 100000)
     (n1, n2) <- Hedgehog.forAll $ (,) <$> gen <*> gen
-    let ghcResult = divMod n1 n2
-        plutusResult = Ratio.divMod n1 n2
+    ghcResult <- tryHard $ divMod n1 n2
+    plutusResult <- tryHard $ Ratio.divMod n1 n2
```

Comment:
Why catch exceptions instead of generating a denominator and testing different things depending on whether the denominator is 0 or not?

Reply:
This way seemed simpler: we get a value which is an `Either`/`Maybe`, and we just check it for equality. That covers both cases uniformly, no?

Reply:
Yeah, but I kinda like testing that output do not only agree with each other, but also with what you'd expect them to be (if the divisor was zero, then the result is `Nothing`). Whatever, I agree it doesn't really matter.